.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+	env: {
+		browser: true,
+		es2021: true,
+		node: true,
+	},
+	extends: ['eslint:recommended', 'plugin:react/recommended'],
+	parserOptions: {
+		ecmaFeatures: {
+			jsx: true,
+		},
+		ecmaVersion: 12,
+		sourceType: 'module',
+	},
+	plugins: ['react'],
+	rules: { indent: ['error', 'tab'] },
```

Comment:
Good job Maxime.Just one small thing, we convined to use two space inndentation not the tab. Plz change the *tab* with 2 as it was